packages/extension-ui/src/components/RadioGroup.tsx

```diff
@@ -28,9 +36,26 @@ const RadioGroup: React.FC<Props> = ({ className, defaultSelectedValue, onSelect
     [onSelectionChange]
   );
 
+  const alephOptions = withTestNetwork
+    ? options.filter((option) => option.text.includes('Aleph Zero'))
+    : options.filter((option) => option.text === 'Aleph Zero');
```

Comment:
Do we need filter here? Isn't `.find` would be enough and stop after we get the Aleph rather than go through all options?